.github/workflows/ci.yml

```diff
@@ -455,8 +455,7 @@ jobs:
   ##############################################################################
   test_shark_model_suite:
     needs: [setup, build_all, build_tf_integrations]
-    # Disabled while failing, see https://github.com/nod-ai/SHARK/issues/373
```

Comment:
For future reference, you can send a PR that reverts another from GitHub's UI, which will reference commits (like a rollback CL): 
![image](https://user-images.githubusercontent.com/4010439/192907754-f2d2bc8c-ad77-47ea-b854-2ed8aa55abcf.png)

(this reverts https://github.com/iree-org/iree/pull/10578)

Reply:
Nice! Thanks for the tip.